.azure/applications/resource-registry-syncer-job/main.bicep

```diff
@@ -0,0 +1,83 @@
+targetScope = 'resourceGroup'
+
+@description('The tag of the image to be used')
+@minLength(3)
+param imageTag string
+
+@description('The environment for the deployment')
+@minLength(3)
+param environment string
+
+@description('The location where the resources will be deployed')
+@minLength(3)
+param location string
+
+@description('The name of the container app environment')
+@minLength(3)
+@secure()
+param containerAppEnvironmentName string
+
+@description('The name of the Key Vault for the environment')
+@minLength(3)
+@secure()
+param environmentKeyVaultName string
+
+@description('The cron expression for the job schedule')
+@minLength(9)
+param jobSchedule string
+
+var namePrefix = 'dp-be-${environment}'
+var baseImageUrl = 'ghcr.io/digdir/dialogporten-'
+var tags = {
+  Environment: environment
+  Product: 'Dialogporten'
+}
+var name = '${namePrefix}-resource-registry-syncer-job'
```

Comment:
Named the job resource-registry-syncer-job.. The command itself is update-subject-resources, so maybe name it something similar?

`update-subject-resources-job`?

Reply:
Yea did a renaming now..!